guide/source/program_administrator/introduction.rst

```diff
@@ -111,4 +113,4 @@ residential.
 
 For more information on BEDES, visit:
 http://energy.gov/eere/buildings/building-energy-data-exchange-specification-bedes
-
+[THIS WEB ADDRESS IS NOT WORKING FOR ME.]
```

Comment:
Weird. According to Google they cached this page on January 12, 2015. Maybe it's just down today. I'll check back later to see if it comes back up.